src/styles/blocks/_catalog.scss

```diff
@@ -2,18 +2,23 @@
   display: flex;
   flex-wrap: wrap;
 
-  max-width: 944px;
-  margin: 0 auto;
-
+  max-width: ($card-width * 4) + ($card-margin-r * 3);
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color $duration linear, transform $duration linear;
```

Comment:
well, IMHO `linear` is strange solution, but okay 🙂 (as for me default `ease` looks better)